### ISSUE TITLE
Fix MUI <select> displaying incorrectly

### ISSUE THREAD

**Issue by RAGessler** [RAGessler/DirSales-CRM#1]
Select is displaying menu items as a horizontal list instead of vertically as intended.

**Comment by RAGessler** [comment]
The issue is being caused by something deep in the DCC starter code I was provided.  I attempted to remove unused imports and rebuild my package & package-lock files as well as my node_modules folder with no improvements. I confirmed that it was the starter code by using a fresh copy and adding the MUI element with the same result. 

I have been wanting to work on a full front-end refactor and maybe even move the backend to a BaaS such as firebase.  This fix will have to wait until then when I start with some fresh code.

The DCC starter code was just a barebones React/Django project to make sure that every student started on the same platform so instructors would be able to help if needed.

**Comment by RAGessler** [comment]
The issue is being caused by something deep in the DCC starter code I was provided. I attempted to remove unused imports and rebuild my package & package-lock files as well as my node_modules folder with no improvements. I confirmed that it was the starter code by using a fresh copy and adding the MUI element with the same result.

I have been wanting to work on a full front-end refactor and maybe even move the backend to a BaaS such as firebase. This fix will have to wait until then when I start with some fresh code.

The DCC starter code was just a barebones React/Django project to make sure that every student started on the same platform so instructors would be able to help if needed.
